### ISSUE TITLE
[IndexBundle] Range Filter does not work if min or max value is 0

### ISSUE THREAD

**Issue by ychanan** [coreshop/CoreShop#1602]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no

If min value (and/or max value) in the range is zero (e.g. 0-500), the RangeFilterConditionProcessor skips applying the condition to the list.

As seen in the `RangeFilterConditionProcessor::addCondition()` method:
```
if (!empty($valueMin) && !empty($valueMax)) {
    $fieldName = $field;

    if ($isPrecondition) {
        $fieldName = 'PRECONDITION_' . $fieldName;
    }

    $list->addCondition(new RangeCondition($field, $valueMin, $valueMax), $fieldName);
}
```

**Comment by dkarlovi** [comment]
Right, I don't see why 0 should be anything special, min can easily be negative, the range should check for `!== null` probably.
